packages/contracts/scripts/shared/helpers/loadEnv.ts

```diff
@@ -7,7 +7,7 @@ export const loadEnv = (path: string): Env => {
   dotenv.config({ path });
   const rpcUrl = process.env.RPC_URL || "http://localhost:8545";
```

Comment:
This should be selected from our RPC selector script.

Reply:
like this @pavlovcik 
![image](https://user-images.githubusercontent.com/109317559/217235331-93ba3981-197f-46c3-a291-8d80f0db87e9.png)

Reply:
You should have just posted it as a suggestion on the comment lol but yes like that. I'll manually copy